paasta_tools/marathon_tools.py

```diff
@@ -188,7 +187,7 @@ def copy(self):
         return self.__class__(self.service, self.instance, dict(self.config_dict), dict(self.branch_dict))
 
     def get_min_instances(self):
-        return self.config_dict.get('instances', 1)
+        return self.config_dict.get('min_instances', 1)
```

Comment:
ty. This is closer in line with AWS autoscaling words which is kinda the "gold standard"